bcda/auth/plugin/alpha_test.go

```diff
@@ -183,22 +189,28 @@ func (s *AlphaAuthPluginTestSuite) TestRequestAccessToken() {
 func (s *AlphaAuthPluginTestSuite) TestRevokeAccessToken() {
 	db := connections["TestRevokeAccessToken"]
 
-	userID, acoID := "EFE6E69A-CD6B-4335-A2F2-4DBEDCCD3E73", "DBBD1CE1-AE24-435C-807D-ED45953077D3"
-	var user models.User
-	db.Find(&user, "UUID = ? AND aco_id = ?", userID, acoID)
-	// Good Revoke test
-	_, tokenString, _ := s.AuthBackend.CreateToken(user)
-
+	const UserID, AcoID = "EFE6E69A-CD6B-4335-A2F2-4DBEDCCD3E73", "DBBD1CE1-AE24-435C-807D-ED45953077D3"
```

Comment:
Capitalizing these const names exports them, right? Do we need that?

Reply:
I don't think variables inside a method are exported. These are constant values (can't be changed or assigned), and there is a convention for indicating that in naming. In C and Java, the convention is all caps, like USER_ID and ACO_ID. In go, the convention is initial cap only. Weird and confusing to me, but it is what it is.